celements-user-manager/web-module/src/main/webapp/resources/celRes/userAdmin/table.css

```diff
@@ -0,0 +1,146 @@
+@charset "UTF-8";
+#userAdmin #searchForm #searchoutput .struct_table {
+  display: flex;
+  flex-flow: column;
+}
+
+#userAdmin #searchForm #searchoutput .struct_table ul.struct_table_header, 
+#userAdmin #searchForm #searchoutput .struct_table ul.struct_table_data {
+  list-style-type: none;
+}
+
+#userAdmin #searchForm #searchoutput .struct_table ul.struct_table_header {
+  padding: 5px 0px;
+  margin: 0px 0px;
+  font-weight: bold;
+}
+
+#userAdmin #searchForm #searchoutput .struct_table ul.struct_table_data {
+  display: flex;
+  flex-flow: column;
+  padding: 0px 0px;
+  margin: 0px 0px;
+}
+
+#userAdmin #searchForm #searchoutput .struct_table ul.struct_table_data li.struct_table_row:nth-child(odd){
+  background-color: var(--dirty-white);
+}
+
+#userAdmin #searchForm #searchoutput .struct_table ul li.struct_table_row {
+  display: flex;
+  flex-flow: row wrap;
+  align-items: flex-start;
+  padding: 2px 0px;
+  margin: 1px 0px;
+}
+
+#userAdmin #searchForm #searchoutput .struct_table ul.struct_table_data li.struct_table_row:hover,
+#userAdmin #searchForm #searchoutput .struct_table ul.struct_table_data li.struct_table_row.open {
+  background-color: var(--lightgrey-color);
+}
+
+#userAdmin #searchForm #searchoutput .struct_table .column_arrow {
+  flex: 0 0 20px;
+  text-align: center;
+}
+
+#userAdmin #searchForm #searchoutput .struct_table li.struct_table_row .column_arrow {
+  -webkit-transition: transform 0.5s linear;
+  -moz-transition: transform 0.5s linear;
+  -o-transition: transform 0.5s linear;
```

Comment:
```suggestion
```
the transition property is widely supported nowadays: https://caniuse.com/?search=transition